components/Header/index.tsx

```diff
@@ -1,14 +1,78 @@
 "use client";
 
-import { Children, ReactNode, useEffect, useState } from "react";
+import { useEffect, useState } from "react";
+import Image from "next/image";
 import Search from "@/components/Search";
-import React from "react";
+import idsLogo from "../../public/assets/images/ids/IDS logo_landscape_WHITE.svg";
+
+const cardListTopicItems = [
```

Comment:
Also a personal preference thing, but if it doesn't breach any JS standards around file structure my instinct would be to move this hardcoded list to another file so the index.tsx focusses more directly on the crux of what it needs to do. If we refactor these items to be pulled from a data source in the future, then that complexity can happen in that other file rather than all in one? 

But if it's a bad JS pattern then ignore me!

Reply:
fwiw I dont feel the same about popularItems, I'm happy for that to stay, its just the topics

Reply:
Moved the topics list into a data folder, I should have done this before as I reuse it a couple times.